### DISCUSSION TITLE
Time Column of Log Window

### DISCUSSION THREAD

**Discussion by PaulA-V** [dannagle/PacketSender#342] [Q&A] [answered]
Hi Dan,

I have been using PacketSender for about a month now.  I am running a 24/7 capture of UDP traffic from a client.  I noticed that when sorting the log window by Time (descending) as the time passes to a new day the entries no longer show up at the top of the log window but rollover to where the last day transition resided in the window.  This makes sense given the Time stamp does not include the Date.  Am I missing something regarding the functionality of the log window?  If not and this is indeed how it's meant to work, may I submit an Idea for adding the Date to the time stamp such that sorting will allow the prepending/appending of the log entries upon day transition?

Regards,

Paul

**Comment by dannagle** [comment, marked as answer]
Hello Paul,

This is a known bug to be fixed in the next release. The next release will include the full date so it stays sorted correctly.

See here: #341

**Reply by PaulA-V** [reply]
Okay thanks Dan!  Sorry I didn't check the bugs prior to posting.


Regards,

Paul
